adsws/accounts/views.py

```diff
@@ -203,6 +204,111 @@ def post(self):
         return {'message': 'success'}, 200
 
 
+
+class UserInfoView(Resource):
+    """
+    Implements getting user info from session ID or token, it should be limited
+    to internal use only
+    """
+    decorators = [
+        ratelimit.shared_limit_and_check("500/43200 second", scope=scope_func),
+        oauth2.require_oauth('adsws:internal')
+    ]
+
+    def post(self):
+        """
+        This endpoint provides the full identifying data associated to a given
+        token (indicated via POST data 'access_token' parameter). Example:
+
+        curl -H 'authorization: Bearer:xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx'
+            -H "Content-Type: application/json"
+            -d '{"acess_token":"xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"}'
+            -X POST
+            'https://dev.adsabs.harvard.edu/v1/accounts/session'
```

Comment:
`https://dev.adsabs.harvard.edu/v1/accounts/session` looks very nice, but app.py suggests the view is registered under one endpoint `/info` 

GET and POST methods have different parameters; if we are trying to compliant - better to just allow GET (becasue it's read only) wit one parameter

`/info/sfsadfsadfsadfs' (token)
`/info/scccccccccccccccccccooooooooooooooookie`
`/info/35` (user_id)